examples/booktest/postgresql/jsonemptyarrays/db_test.go

```diff
@@ -0,0 +1,40 @@
+// +build examples
```

Comment:
Any opinions about this being nested under `./postgresql/json` vs being it's own top level test dir?